src/plugins/PluginAPI.hpp

```diff
@@ -53,7 +53,7 @@ struct SVersionInfo {
     std::string commits;
 };
 
-#define APICALL extern "C"
+#define APICALL extern "C++"
```

Comment:
I'm also not confident on this change, but it seems right to me. These functions definitely aren't C ABI compatible because they have all sorts of C++ types like `std::string`, so I think this is the correct change, but I'm not 100% sure what these extern function modifiers are doing.

Reply:
this is wrong. The extern C is to disable name mangling